src/pages/Tasks/components/TaskAccordionEditing/index.js

```diff
@@ -0,0 +1,78 @@
+import PropTypes from "prop-types";
+import {
+  Accordion,
+  AccordionDetails,
+  AccordionSummary,
+  Button,
+  TextField,
+} from "@mui/material";
+
+import { useForm } from "hooks";
+
+import styles from "./style.module.scss";
+
+const TaskAccordionEditing = ({
+  id,
+  taskValue,
+  isEditing,
+  handleEditingTaskCancel,
+  handleEditingTaskConfirm,
+}) => {
+  const { text, description } = taskValue;
+  const { form, handleChange } = useForm({ text, description });
+  return (
+    <div className={styles.wrapper}>
+      <Accordion defaultExpanded={isEditing} expanded={true}>
+        <AccordionSummary>
+          <TextField
+            name="text"
+            label="Task title edit"
+            variant="outlined"
+            value={form.text}
+            onChange={handleChange}
+            sx={{ width: "100%" }}
+          />
```

Comment:
Better to create custom component instead of using TextField directly from Material UI